### PR TITLE
Script to import berlin geo data

### PR DESCRIPTION
**Pull request by 2e2a**
based on https://github.com/liqd/adhocracy3/blob/master/src/adhocracy_meinberlin/adhocracy_meinberlin/scripts/ad_import_geodata.py

### CODE REVIEW

File: apps/maps/management/commands/import_geodata.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            if not map_models.MapPreset.objects.filter(name=name).exists():
+                self._create_map_preset(name, feature, category)
+        os.remove(tmpfile)
+
+    def _preset_category(self, name):
+        try:
+            category = map_models.MapPresetCategory.objects.get(name=name)
+        except map_models.MapPresetCategory.DoesNotExist:
+            category = map_models.MapPresetCategory(name=name)
+            category.save()
```
Comment (vellip) [inline apps/maps/management/commands/import_geodata.py]: What about using `get_or_create` instead here? https://docs.djangoproject.com/en/1.11/ref/models/querysets/#get-or-create

**Comment by vellip** [comment]
LGTM

File: apps/maps/management/commands/import_geodata.py (hunk 1 above)
Comment (2e2a) [inline apps/maps/management/commands/import_geodata.py]: Thanks, using this.

**Comment by 2e2a** [comment]
Updated ordering and category structure.

**Pull request merged**